assets/style/index.scss

```diff
@@ -0,0 +1,290 @@
+$color_1: hsl(30, 10%, 34%);
+$color_2: hsl(24, 5%, 18%);
+$color_3: hsl(332, 51%, 32%);
+$color_4: hsl(14, 45%, 36%);
+$font-family_1: "Outfit", serif;
+$font-family_2: "Young Serif", serif;
+$background-color_1: hsl(30, 54%, 90%);
+$background-color_2: hsl(0, 0%, 100%);
+$background-color_3: hsl(330, 100%, 98%);
```

Comment:
we can use more descriptive naming for colors, for example primary color, secondary color, primary color-light, primary-color-dark, navbar-background-color, font-color ...etc